playwright-tests/tests/profile.spec.js

```diff
@@ -0,0 +1,174 @@
+import { expect, test } from "@playwright/test";
+import { ROOT_SRC } from "../util/constants";
+import path from "path";
+
+test.describe("?page=profile", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}?page=profile`);
+  });
+
+  test.describe("User logged in", () => {
+    test.beforeEach(async ({ page }) => {
+      // Intercept IPFS requests
+      await page.route("**/add", async (route) => {
+        const modifiedResponse = {
```

Comment:
This is much better!! Great tests, good separation and structure. Well done.

I think you forgot to remove this here, L13- L18, since you already correctly moved it to the test that uses it, on L62